packages/react/src/index.scss

```diff
@@ -9,5 +9,5 @@
 @import './components/Button';
 
 * {
-  font-family: var(--c--theme--typo--font-base);
+  font-family: var(--c--theme--typo--families--base);
```

Comment:
When I see this change, I realized token names should stick to CSS Property name.
For example, in this case, I would rename token :
`--c--theme--font-family--base)`